packages/venia-ui/lib/RootComponents/CMS/cms.js

```diff
@@ -1,8 +1,42 @@
-import React from 'react';
-import CategoryList from '../../components/CategoryList';
+import React, { useEffect } from 'react';
+import { useLazyQuery } from '@apollo/react-hooks';
+import cmsPageQuery from '../../queries/getCmsPage.graphql';
+import { fullPageLoadingIndicator } from '../../components/LoadingIndicator';
+import RichContent from '../../components/RichContent';
 
-const CMS = () => {
-    return <CategoryList title="Shop by category" id={2} />;
+const CMSPage = props => {
+    const { id } = props;
+    const [runQuery, queryResponse] = useLazyQuery(cmsPageQuery);
```

Comment:
Why `lazyQuery`?

Reply:
Updated to use `useQuery`: https://github.com/magento/pwa-studio/pull/1872/commits/c11d5364191e687cfc394f354c3a12740dea1aef